pages/common/mmv.md

```diff
@@ -0,0 +1,19 @@
+# mmv
+
+> Move and rename files in bulk.
+
+- Rename all files with a certain extension to a different extension:
+
+`mmv "*{{.old_extension}}" "#1{{.new_extension}}"`
+
+- Copy report6part4.txt to ./french/rapport6partie4.txt along with all similarly named files:
```

Comment:
I feel we can do better with the proposed example. This stuff is _vague_. Not an easy comment to work with, I know. I got so distracted by the example that I didn't see immediately that the keyword here is "copy".